documentation/manual/src/main/paradox/interpreters/pekko-http.md

```diff
@@ -0,0 +1,119 @@
+# Pekko HTTP
+
+Client and server backed by [Pekko HTTP](https://pekko.apache.org/docs/pekko/current/).
+
+@@@ warning
+
+As explained in the section
+[Mixed versioning is not allowed](https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed)
+in the Pekko documentation, you have to make sure that all the
+Pekko modules of your application have the same version.
+
+For this reason, the interpreters `pekko-http-server` and
+`pekko-http-client` have marked their dependency to Pekko as “provided”.
+
+As a consequence, to use these interpreters you will have to
+explicitly add a dependency on `pekko-stream`:
+
+~~~ scala
+"org.apache.pekko" %% "pekko-stream" % "<pekko-version>"
+~~~
+
+Where `<pekko-version>` is binary compatible and higher or equal to @var[pekko-version].
+
+@@@
+
+## Client
+
+@coordinates[pekko-http-client]
+
+@scaladoc[API documentation](endpoints4s.pekkohttp.client.index)
+
+### `endpoints4s.pekkohttp.client.Endpoints`
+
+The `Endpoints` interpreter fixes the `Endpoint[A, B]` type to a function
+from `A` to `Future[B]`:
+
+@@snip [Endpoints.scala](/pekko-http/client/src/main/scala/endpoints4s/pekkohttp/client/Endpoints.scala) { #endpoint-type }
+
+This means that, given the following endpoint definition:
+
+@@snip [EndpointsDocs.scala](/pekko-http/client/src/test/scala/endpoints4s/pekkohttp/client/EndpointsDocs.scala) { #endpoint-definition }
+
+It can be invoked as follows:
+
+@@snip [EndpointsDocs.scala](/pekko-http/client/src/test/scala/endpoints4s/pekkohttp/client/EndpointsDocs.scala) { #invocation }
+
+### `endpoints4s.pekkohttp.client.ChunkedEntities`
+
+The `ChunkedEntities` interpreter fixes the `Chunks[A]` type to `pekko.stream.scaladsl.Source[A, _]`:
+
+@@snip [ChunkedEntities.scala](/pekko-http/client/src/main/scala/endpoints4s/pekkohttp/client/ChunkedEntities.scala) { #stream-type }
+
+This means that, given the following endpoint definition:
+
+@@snip [ChunkedEntitiesDocs.scala](/algebras/algebra/src/test/scala/endpoints4s/algebra/ChunkedEntitiesDocs.scala) { #streamed-endpoint }
+
+It can be invoked as follows:
+
+@@snip [ChunkedEntitiesDocs.scala](/pekko-http/client/src/test/scala/endpoints4s/pekkohttp/client/ChunkedEntitiesDocs.scala) { #invocation }
+
+## Server
+
+@coordinates[pekko-http-server]
+
+@scaladoc[API documentation](endpoints4s.pekkohttp.server.index)
+
+### `endpoints4s.pekkohttp.server.Endpoints`
+
+The `Endpoints` interpreter fixes the `Endpoint[A, B]` type to something that,
+given an implementation function `A => B`, returns an `pekko.http.scaladsl.server.Route`
```

Comment:
```suggestion
given an implementation function `A => B`, returns a `pekko.http.scaladsl.server.Route`
```

BTW, should it be `org.apache.pekko` instead of `pekko`?